Packs/TrendMicroVisionOne/Integrations/TrendMicroVisionOneV3/TrendMicroVisionOneV3.py

```diff
@@ -1612,46 +2044,60 @@ def download_investigation_package(
     Downloads the Investigation Package of the specified object based on
     submission id and sends the file to demist war room where it can be downloaded.
     :type client: ``Client``
-    :param client: client object to use http_request.
+    :param client: client object used to initialize pytmv1 client.
     :type args: ``dict``
     :param args: args object to fetch the argument data.
     :return: sends data to demisto war room.
     :rtype: ``dict`
     """
-    submission_id = args.get(SUBMISSION_ID)
-    file_name = args.get(FILE_NAME)
-    # If a file name is not provided, default value of
-    # Sandbox_Investigation_Package is set for the package.
-    if not file_name:
-        file_name = "Sandbox_Investigation_Package.zip"
-    query_params: Dict[str, Any] = {}
-    token = client.api_key
-    headers = {AUTHORIZATION: f"{BEARER} {token}"}
-    response = requests.get(
-        f"{client.base_url}"
-        + DOWNLOAD_INVESTIGATION_PACKAGE.format(submissionId=submission_id),
-        params=query_params,
-        headers=headers,
+    # Required Params
+    submit_id = args.get(SUBMISSION_ID, EMPTY_STRING)
+    # Optional Params
+    poll = args.get(POLL, TRUE)
+    poll_time_sec = args.get(POLL_TIME_SEC, 0)
+    file_name = args.get(FILE_NAME, EMPTY_STRING)
```

Comment:
the same here, `FILE_NAME` not defined in the YML and be reassigned in line 2064

Reply:
corrected.